lua/oil/config.lua

```diff
@@ -80,6 +80,19 @@ local default_config = {
     win_options = {
       winblend = 10,
     },
+    -- This is the config that will be passed to nvim_open_win.
+    -- Change values here to customize the layout
+    -- For example, you can use this to position the window on the right
+    -- override = function(conf)
+    --   local opts = {
+    --     col = vim.o.columns - conf.width,
+    --     zindex = 80,
+    --   }
+    --   return vim.tbl_deep_extend("force", conf, opts)
+    -- end,
```

Comment:
I actually think you don't need the example here. This example doesn't really do much visually, and any example that produces a sufficiently differentiated layout is probably going to be long enough that it would better fit in a "recipes" section of the docs.